pkgs/by-name/li/lianad/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+, systemd
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "lianad";
+  version = "5.0"; # keep in sync with liana
+
+  src = fetchFromGitHub {
+    owner = "wizardsardine";
+    repo = "liana";
+    rev = "v${version}";
+    hash = "sha256-RkZ2HSN7IjwN3tD0UhpMeQeqkb+Y79kSWnjJZ5KPbQk=";
+  };
+
+  cargoSha256 = "sha256-6r8G4sO/aDO5VyJbmbdFz3W2b8ZmulXJZtnz8IAwcaU=";
+
+  buildInputs = [ systemd ];
+
+  postInstall = ''
+    install -Dm0644 ./contrib/lianad_config_example.toml $out/etc/liana/config.toml
```

Comment:
Are you sure it will read this (without wrapping with some `XDG_` environment variables?)

Reply:
can you elaborate? my testing rationale is based on the contents of `result` after running `nix-build -A lianad`:

```
result
├── bin
│   ├── liana-cli
│   └── lianad
└── etc
    └── liana
        └── config.toml
```